src/guides/v2.3/payments-integrations/braintree/braintree.md

```diff
@@ -0,0 +1,114 @@
+---
+group: payments-integrations
+title: Prevent braintree card attack
+contributor_name: Ziffity
+contributor_link: https://www.ziffity.com/
+--- 
+
+Reference to [Carding Attack #28614](https://github.com/magento/magento2/issues/28614) Gene Commerce has come up with latest version [GitHub](https://github.com/genecommerce/module-braintree-magento2/releases/tag/4.0.7)
+
+### Requirements
+
+**Magento** | **PHP** | **Braintree**
+--- | --- | ---
+`2.3,2.4` | `> 7.x` | `>= 4.0.7`
+
+### How to prevent braintree card bot attack ?
+
+1. Bot attacker will create the order several times using rest endpoint `POST <host>/rest/<store_code>/V1/guest-carts/<cart-id>/payment-information`.
+
+1. To avoid card bot attack, we need to enable the default `Magento` & `Braintree` reCaptcha configuration from admin.
```

Comment:
```suggestion
1. To avoid the card bot attack, it is necessary to enable the default `Magento` & `Braintree` reCaptcha configuration from admin.
```

Reply:
Spelling correction updated & committed...!